src/realm.js

```diff
@@ -75,6 +75,10 @@ async function loadModels(Spine, models, opts) {
   const schemaInfo = await Spine.driver.querySchemaInfo(database, tables);
 
   for (const model of models) {
+    // assign driver if model's driver not exist
+    if (!model.driver) model.driver = Spine.driver;
+    // assign options if model's options not exist
+    if (!model.options) model.options = Spine.options;
```

Comment:
was `Spine.driver` initialized elsewhere already?

Reply:
`Spine.driver` initialized in `new Realm(options)`,  but `model.driver/options` may not be initialized,  because `model` may not extended from the `Spine` aforementioned, this is an edge case.